src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,72 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+    static String Delimiter = ",|:";
+
+    public static int sum(String value) {
+        if(isBlankOrisNull(value))
+            return 0;
```

Comment:
지금은 0을 반환하지만, 제약에따라 다른 값을 반환해야 할 수도 있는데, 그럴 경우 갑자기 return xxx;가 될 때 이게 뭐지? 라는 생각이 들 수 있어요. 이런 매직넘버는 상수로 분리해주시면 가독성을 높힐 수 있습니다 👍 

```suggestion
            return DEFAULT_RETURN_VALUE;
```